### ISSUE TITLE
Error in printing Flow TypeAlias 

### ISSUE THREAD

**Issue by amasad** [6to5/6to5#513]
Type alias in the form: `type T = Y;` fails. See example below:

```
amasad-mbp:client amasad$ 6to5
type x = number;
^D
/usr/local/lib/node_modules/6to5/lib/6to5/util.js:254
    throw err;
          ^
Error: stdin: Unknown node type TypeAlias.
    at Controller.traverse (/usr/local/lib/node_modules/6to5/node_modules/estraverse/estraverse.js:513:31)
    at traverse (/usr/local/lib/node_modules/6to5/node_modules/estraverse/estraverse.js:705:27)
    at Object.attachComments (/usr/local/lib/node_modules/6to5/node_modules/estraverse/estraverse.js:794:9)
    at Object.exports.parse (/usr/local/lib/node_modules/6to5/lib/6to5/util.js:230:16)
    at File.parse (/usr/local/lib/node_modules/6to5/lib/6to5/file.js:237:15)
    at Object.transform (/usr/local/lib/node_modules/6to5/lib/6to5/transformation/transform.js:10:15)
    at Object.exports.transform (/usr/local/lib/node_modules/6to5/bin/6to5/util.js:23:20)
    at ReadStream.<anonymous> (/usr/local/lib/node_modules/6to5/bin/6to5/file.js:81:25)
    at ReadStream.emit (events.js:117:20)
    at _stream_readable.js:943:16
```

**Comment by sebmck** [comment]
Doesn't look like I added flow type visitor keys. I'll add them now.

**Comment by sebmck** [comment]
Fixed in 2.12.6. Note that 6to5 currently doesn't have support for printing type definitions in the generated output although it could possibly be added, see d68f1e99105a8c532097dfc7e1f434753f8c083e.

**Comment by amasad** [comment]
It's fine, we only need them stripped for now. (This might change in the
future but this is the correct way for now)
